lib/byron/test/data/cardano-node/node.config

```diff
@@ -12,7 +12,7 @@ NodeId:
 NumCoreNodes: 1
 PBftSignatureThreshold: 1
 Protocol: RealPBFT
-GenesisFile: genesis.json
+GenesisFile: test/data/cardano-node/genesis.json
```

Comment:
Does this work from nix? (the previous `genesis.json` was probably overridden by the CLI arg, and probably didn't work)

Reply:
as far as I understand cardano-node team decided to point to genesis file and resign from `genesis-file` and `genesis-hash` option. So I take `genesis.yaml` transform it (with adding `startTime`) and it always lands in this directory. It does not work with just tmp directory for me. I do not know if it works with nix, though. Feel free to experiment and push on top

Reply:
addressed in https://github.com/input-output-hk/cardano-wallet/pull/1506/commits/c1b1c8e42e57016eb3e42d3837974ca698fc6142